lib/components/Input/password-field/password-field.scss

```diff
@@ -0,0 +1,12 @@
+.password_field {
+
+    div[data-lastpass-icon-root]  {
+       display: none !important;
+    }
```

Comment:
Let's remove and let clients disable lastpass from their side if it's disrupting them